ColorBox: add a Getting Started tutorial (#4348)
DevExpress/devextreme-documentation: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md

> Users can change the value after they pick a shade, or after they click the **OK** button. To specify the mode, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle the value change when users select a color in the drop-down editor or type it in.

Reviewer: ```suggestion
Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code in.
```

> Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code in.

Reviewer 2: ```suggestion
Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code.
```

> The following code displays a notification every time users change the ColorBox value. The background color of the notification is the color they picked. 

Reviewer: ```suggestion
The following code displays a notification every time users change the ColorBox value. The notification's background assumes the picked color. 
```

> The following code displays a notification every time users change the ColorBox value. The notification's background assumes the picked color. 

Reviewer 2: `The notification's background assumes the picked color.`

Can we say the following instead?

`The selected color is applied to the notification's background.`

Author: Yes, I changed it

Reviewer 2: Thank you!

> The following code displays a notification every time users change the ColorBox value. The notification's background assumes the picked color. 

Author: ```suggestion
The following code displays a notification every time users change the ColorBox value. The selected color is applied to the notification's background.
```
